Dockerfile.builder

```diff
@@ -5,14 +5,14 @@ RUN go get -d github.com/tdewolff/minify/cmd/minify \
  && go build -v -o /minify github.com/tdewolff/minify/cmd/minify
 
 # Set the version of Github Pages to use for each docs archive
-FROM starefossen/github-pages:latest
+FROM starefossen/github-pages:177
```

Comment:
sgtm, but surprised if `:latest` would be older than `:177` 🤔 
possibly the image wasn't rebuilt for a while, so being explicit is good to do

Reply:
It was last rebuilt two months ago, so it's at v173 or so. The problems with using `latest` are listed in the changelog message; in particular it's never going to change => no clean way to trigger an auto-rebuild (surely we can add a comment or fix a typo but that is oh so wrong thing to do).